js/pages/questionnaire.js

```diff
@@ -78,7 +78,7 @@ async function startModule(data, modules, moduleId, questDiv) {
             url += "/" + path;
             
             let moduleText = await (await fetch(url)).text();
-            let match = moduleText.match("{\"version\": \"([0-9\.]+)\"}");
+            let match = moduleText.match("{\"version\":\"([0-9\.]+)\"}");
```

Comment:
This regex can match other strings we don't want (eg '{"version":"1.2.3.4.5.6.7..."}')